fe/fe-core/src/main/java/com/starrocks/sql/analyzer/InsertAnalyzer.java

```diff
@@ -243,7 +243,9 @@ public static void analyze(InsertStmt insertStmt, ConnectContext session) {
         insertStmt.setTargetTable(table);
         insertStmt.setTargetPartitionIds(targetPartitionIds);
         insertStmt.setTargetColumns(targetColumns);
-        session.getDumpInfo().addTable(database.getFullName(), table);
+        if (session.getDumpInfo() != null) {
```

Comment:
why check this? 
why not check this before ?

Reply:
Before queries always put the related tables/views/... into DumpInfo.

I've changed this to only save related infos when it's from HTTP query dump or `enable_query_dump` is true.  I think this is more reasonable.